### DISCUSSION TITLE
Dịch Manifold Learning

### DISCUSSION THREAD

**Discussion by quangnhat185** [mlbvn/handson-ml2-vn#57] [Terminology] [answered]
Trong chương 8 có thuật ngữ `Manifold Learning` nói đến phương pháp học bằng cách trải phẳng theo cuộn tập dữ liệu trong không 3D thành không gian 2D. 

Mình search thì thấy có nguồn [Wiki](https://vi.wikipedia.org/wiki/%C4%90a_t%E1%BA%A1p) dịch `Manifold` là `Đa tạp` , nhưng chưa thấy cách dịch này phổ biến lắm. Mọi người cho ý kiến nhé?

cc: @mlbvn/handson-ml2 

### Context: 

![image](https://user-images.githubusercontent.com/41325216/111151161-5fdbf400-858f-11eb-9f02-7b9f7b7721c6.png)

**Comment by quangvnai** [comment, marked as answer]
Interested! :))
Thấy từ điển trong hình học thì dịch `đa tạp`. Nghe không quen tai cho lắm.
http://tratu.soha.vn/dict/en_vn/Manifold

**Reply by rootonchair** [reply]
Em nghĩ nếu đã định nghĩa rồi thì cứ dùng thôi. Em cũng thấy tên `đa tajp` trên một vài giáo trình của các trường đại học

**Reply by lkhphuc** [reply]
Mình cũng thấy `đa tạp` OK. Từ này mượn thẳng trong toán, mà bên toán dịch vậy nên mình cứ theo.
